Journeys not being called

Hi there, 

I'm implementing synthetics using the heartbeat Docker image but my journey is not being called by the elastic module. 
All my files are being placed in `/usr/share/heartbeat/suites`, with name `xxxx.journey.js` but they are not executed. 

My manifest file look like this:
```yml
heartbeat.monitors:
- type: browser
  name: Accounts
  id: accounts-auth
  schedule: "@every 1m"
  source:
    local:
      path: "${SUITES_DIR}"
```

docker-compose.yml:
```yml
version: '3.2'

services:
  heartbeat:
    build:
      context: .
    command: --strict.perms=false -c /etc/heartbeat.yml -e
    env_file:
     - .env
    volumes:
      - type: bind
        source: ./manifests/development/heartbeat.yml
        target: /etc/heartbeat.yml
      - "./dist:/usr/share/heartbeat/suites"
    networks:
      - docker-elk_elk

networks:
  docker-elk_elk:
    external: true

```

I'm unable to deploy my synthetics tests because of it. Am I doing anything wrong?

I'm receiving this message from the docker logs:

```bash
heartbeat_1  | 2021-06-30T20:18:17.447Z INFO    browser/browser.go:39   Synthetic monitor detected! Please note synthetic monitors are an experimental unsupported feature!
heartbeat_1  | 2021-06-30T20:18:17.716Z INFO    synthexec/synthexec.go:128      Running command: /tmp/elastic-synthetics-046899047/node_modules/.bin/elastic-synthetics . --screenshots --json --network --outfd 3 in
 directory: '/tmp/elastic-synthetics-046899047'
heartbeat_1  | 2021-06-30T20:18:17.933Z INFO    synthexec/synthexec.go:224      stderr: error: unknown option '--json'
heartbeat_1  | 2021-06-30T20:18:17.936Z ERROR   synthexec/synthexec.go:172      Error waiting for command /tmp/elastic-synthetics-046899047/node_modules/.bin/elastic-synthetics . --screenshots --json --network --o
utfd 3: exit status 1
heartbeat_1  | 2021-06-30T20:18:17.936Z INFO    synthexec/synthexec.go:176      Command has completed(1): /tmp/elastic-synthetics-046899047/node_modules/.bin/elastic-synthetics . --screenshots --json --network --o
utfd 3
heartbeat_1  | 2021-06-30T20:18:17.936Z WARN    synthexec/synthexec.go:183      Error executing command '/tmp/elastic-synthetics-046899047/node_modules/.bin/elastic-synthetics . --screenshots --json --network --ou
tfd 3': exit status 1
```

--- a/__tests__/cli.test.ts
+++ b/__tests__/cli.test.ts
@@ -41,19 +41,21 @@ describe('CLI', () => {
     expect(await cli.exitCode).toBe(1);
   });
 
-  it('produce json output via --json flag', async () => {
-    const cli = new CLIMock([
-      join(FIXTURES_DIR, 'fake.journey.ts'),
-      '--reporter',
-      'json',
-    ]);
-    await cli.waitFor('fake journey');
-    const output = cli.output();
-    expect(JSON.parse(output).journey).toEqual({
+  it('produce json output  --json and reporter=json flag', async () => {
+    const output = async args => {
+      const cli = new CLIMock([join(FIXTURES_DIR, 'fake.journey.ts'), ...args]);
+      await cli.waitFor('fake journey');
+      expect(await cli.exitCode).toBe(0);
+      return JSON.parse(cli.output());
+    };
+    expect((await output(['--reporter', 'json'])).journey).toEqual({
       id: 'fake journey',
       name: 'fake journey',
     });
-    expect(await cli.exitCode).toBe(0);
+    expect((await output(['--json'])).journey).toEqual({
+      id: 'fake journey',
+      name: 'fake journey',
+    });
   });
 
   it('mimick heartbeat with `--rich-events` flag', async () => {

--- a/src/parse_args.ts
+++ b/src/parse_args.ts
@@ -38,6 +38,7 @@ program
     'configuration path (default: synthetics.config.js)'
   )
   .option('-s, --suite-params <jsonstring>', 'suite variables', '{}')
+  .option('-j, --json', 'output newline delimited JSON')
   .addOption(
     new Option('--reporter <value>', `output repoter format`).choices(
       Object.keys(reporters)

